### ISSUE TITLE
Page title context link lacks correct focus style

### ISSUE THREAD

**Issue by andysellick** [alphagov/govuk_publishing_components#1030]
The context link option for the Page title component does not have a focus state set.

<img width="779" alt="Screen Shot 2019-08-07 at 13 27 10" src="https://user-images.githubusercontent.com/861310/62681768-5b625900-b9b2-11e9-8396-b055a7ae5dbc.png">

This is particularly bad if the 'inverse' option is also called.

<img width="694" alt="Screen Shot 2019-08-07 at 13 28 44" src="https://user-images.githubusercontent.com/861310/62681789-69b07500-b9b2-11e9-8875-8618767d255f.png">

This issue should be addressed after [the V3 PR](https://github.com/alphagov/govuk_publishing_components/pull/1010) is merged.
